### ISSUE TITLE
File is not a zip file Error when loading the pretrained model

### ISSUE THREAD

**Issue by Yichen-fqyd** [nlp-uoregon/trankit#17]
Hi I was trying the customized ner tutorial [notebook](https://github.com/nlp-uoregon/trankit/blob/master/examples/colab/trankit_ner_GermEval14.ipynb)

When I ran code 
```
trankit.verify_customized_pipeline(`
    category='customized-ner', # pipeline category
    save_dir='./save_dir_filtered' # directory used for saving models in previous steps
)
```
It printed "Customized pipeline is ready to use". However when I loaded the pipeline as the instruction, it kept reporting error message: 
/usr/local/lib/python3.7/dist-packages/trankit/utils/base_utils.py in download(cache_dir, language, saved_model_version, embedding_name)
BadZipFile: File is not a zip file.

Can you help me to figure out what did I miss, and how to fix this?

**Comment by minhhdvn** [comment]
Hi @Yichen-fqyd,
Thanks for letting us know.
Could you provide the file structure of your `./save_dir_filtered`?
This command `tree ./save_dir_filtered` might help you do it.

**Comment by Yichen-fqyd** [comment]
`
./save_dir_filtered
./save_dir_filtered/customized-ner
./save_dir_filtered/customized-ner/customized-ner.ner-vocab.json
./save_dir_filtered/customized-ner/customized-ner.ner.mdl
./save_dir_filtered/customized-ner/customized-ner.tagger.mdl
./save_dir_filtered/customized-ner/customized-ner.vocabs.json
./save_dir_filtered/customized-ner/customized-ner_lemmatizer.pt
./save_dir_filtered/xlm-roberta-base
./save_dir_filtered/xlm-roberta-base/customized-ner
./save_dir_filtered/xlm-roberta-base/customized-ner/xlm-roberta-base/customized-ner/customized-ner.zip
./save_dir_filtered/xlm-roberta-base/customized-ner/customized-ner.zip
`
Here is the file structure...


Another question, when I was training the customized pipeline, it will verify all the 5 following files(tagger, vocabs.json, tokenizer, ner_lemmatizer.pt, ner.mdl, ner-vocab.json), but the customized files only have customized-ner.ner.mdl  customized-ner.ner-vocab.json, how to get these other required files?
./customized-ner/customized-ner.tagger.mdl
./customized-ner/customized-ner.vocabs.json
./customized-ner/customized-ner_lemmatizer.pt


Thanks a lot!!

**Comment by minhhdvn** [comment]
Hi @Yichen-fqyd,
If the language of your customized pipeline is one of the [languages](https://trankit.readthedocs.io/en/latest/pkgnames.html#pretrained-languages-their-code-names) that we already have pretrained pipelines, you can obtain the missing files by downloading them from our site. For example, if the language is Chinese, you can do it as below: 
1. If you're using XLM-Roberta base:
```
wget http://nlp.uoregon.edu/download/trankit/v1.0.0/xlm-roberta-base/chinese.zip
```
2. If you're using XLM-Roberta large:
```
wget http://nlp.uoregon.edu/download/trankit/v1.0.0/xlm-roberta-large/chinese.zip
```
Then, you can unzip the downloaded file, rename the extracted files to make them have the prefix `customized-ner`.
Next, you need to move the directory `./save_dir_filtered/customized-ner` to the new location:
1. If you're using XLM-Roberta base:
```
mv save_dir_filtered/customized-ner/ save_dir_filtered/xlm-roberta-base/
```
2. If you're using XLM-Roberta large:
```
mv save_dir_filtered/customized-ner/ save_dir_filtered/xlm-roberta-large/
```
Then I think you're ready to load your customized pipeline.
Thanks.

**Comment by Yichen-fqyd** [comment]
Thank you for the instruction. 
1. I am wondering is it possible to fine tune the model by my own data? for the base model?
2. For the zip file issue, turns out it will download the zip file https://github.com/nlp-uoregon/trankit/blob/fc2c77fde2430109f971f35aa1061d17ac8222ca/trankit/pipeline.py#L70, the the download link [line_65](https://github.com/nlp-uoregon/trankit/blob/fc2c77fde2430109f971f35aa1061d17ac8222ca/trankit/utils/base_utils.py#L65) url is (http://nlp.uoregon.edu/download/trankit/v1.0.0/xlm-roberta-base/customized-ner.zip), which is not exist, so that the downloaded zip file is an error.  If the file is already checked, why do you need to download from url? It seems there are some inconsistency between the verify_customized_pipeline checked dir versus the actually loading directory for Pipeline.

Thank you

**Comment by minhhdvn** [comment]
Hi @Yichen-fqyd and @davidsuba,
Thanks for the comments.
@Yichen-fqyd, you're right that there is a path inconsistency problem. 
We're fixing it and let you know when it's done.

**Comment by minhhdvn** [comment]
Hi @Yichen-fqyd and @davidsuba,
We have fixed the issue in the new version `v1.1.0` of trankit. The new version can be installed via pip with the command: `pip install trankit==1.1.0`.
Thank you again for letting us know this issue.
